### PR TITLE
pass app specific params though flags/envs instead of config map

### PR DESCRIPTION
**Pull request by alavrovinfb**
- [X] added app specific flags
- [X] removed legacy configuration via config map
- [X] updated helm charts

### CODE REVIEW

File: internal/initialize/initial.go [hunk 1]
```diff
@@ -56,16 +61,17 @@ func Initialize(ctx context.Context, logger *logrus.Logger, sig chan os.Signal)
...
 		logger.Fatalf("err while getting pid %s", err)
 	}
 	if pid == 0 {
-		logger.Fatalf("proc with name %s not exist", dataFromK8sAPI.ConfigMapData.ProcName)
+		logger.Fatalf("proc with name %s not exist", procName)
 	}
 
 	//Getting the path to the monitoring directory
-	dirPath := "../proc/" + strconv.Itoa(pid) + "/root/" + dataFromK8sAPI.ConfigMapData.MountPath
+	dirPath := "../proc/" + strconv.Itoa(pid) + "/root/" + viper.GetString("monitoring-path")
```
Comment (alexwatcher) [inline internal/initialize/initial.go]: may be better to use fmt.Sprinf ?

Comment (alavrovinfb) [reply internal/initialize/initial.go]: done

**Review by Kaborda-Irina-ScienceSoft — APPROVED** [review]

**Review by alexwatcher — APPROVED** [review]

File: internal/initialize/initial.go [hunk 2]
```diff
@@ -34,6 +35,9 @@ func init() {
 	fsSum.String("algorithm", algorithm, "hashing algorithm for hashing data")
 	fsSum.String("cm-name", configMapNameForHasher, "name of configMap for hasher")
```
Comment (siarhei-shliayonkin) [inline internal/initialize/initial.go:36]: The configMap integrity-sum-config was removed from the deployment, correct? Then this flag is no longer needs.

File: internal/initialize/initial.go [hunk 3]
```diff
@@ -34,6 +35,9 @@ func init() {
...
 	fsSum.String("main-process-name", mainProcessName, "the name of the main process to be monitored by the hasher")
+	fsSum.String("process", procToMonitor, "the name of the process to be monitored by the hasher")
+	fsSum.String("monitoring-path", pathToMonitor, "the service path to be monitored by the hasher")
```
Comment (siarhei-shliayonkin) [inline internal/initialize/initial.go:39]: Naming - ?
- "main-process-name" - it's a key to search, right? If so, the usage string doesn't match. 
- "process", "monitoring-path" - are not consistent. Maybe, "monitoring-process", "monitoring-path"?

Comment (siarhei-shliayonkin) [reply internal/initialize/initial.go:39]: or even: "monitoring-process", "monitoring-process-path"

File: internal/initialize/initial.go (hunk 2 above)
Comment (alavrovinfb) [reply internal/initialize/initial.go:36]: Good catch, will remove

File: internal/initialize/initial.go (hunk 3 above)
Comment (alavrovinfb) [reply internal/initialize/initial.go:39]: > "main-process-name" - it's a key to search, right? If so, the usage string doesn't match. 
 
Purposes of this flag `main-process-name` is not clear, it seems, it ain't useful at all. Anyway, monitoring process name is passed via `process` flag.

> "process", "monitoring-path" - are not consistent. Maybe, "monitoring-process", "monitoring-path"?

Longer names are not always more descriptive, so I think that current flag names "process", "monitoring-path" are good fit for this context.

**Pull request merged**